service/service_construction.go

```diff
@@ -670,54 +965,102 @@ func (s ConstructionService) createOperationDescription(
 	}
 }
 
+func (s ConstructionService) createOperationDescriptionBridgeUnwrap(
+	currency *types.Currency,
+) []*parser.OperationDescription {
+	var descriptions []*parser.OperationDescription
```

Comment:
why not return this object inline instead of creating a new arr and appending?

```golang
return []*parser.OperationDescription{
   {
		Type: mapper.OpErc20Burn,
		Account: &parser.AccountDescription{
			Exists: true,
		},
		Amount: &parser.AmountDescription{
			Exists:   true,
			Sign:     parser.NegativeAmountSign,
			Currency: currency,
		},
	},
}
```

Reply:
#188